Sources/XcodeGenKit/SettingsBuilder.swift

```diff
@@ -147,14 +147,21 @@ extension SettingsPresetFile {
         if let group = settingPresetSettings[path] {
             return group
         }
-        let relativePath = "SettingPresets/\(path).yml"
-        let possibleSettingsPaths: [Path] = [
-            Path(relativePath),
-            Path(Bundle.main.bundlePath) + relativePath,
-            Path(Bundle.main.bundlePath) + "../share/xcodegen/\(relativePath)",
+        let bundlePath = Path(Bundle.main.bundlePath)
+        let relativePath = Path("SettingPresets/\(path).yml")
+        var possibleSettingsPaths: [Path] = [
+            relativePath,
+            bundlePath + relativePath,
+            bundlePath + "../share/xcodegen/\(relativePath)",
             Path(#file).parent().parent().parent() + relativePath,
         ]
 
+        if let symlink = try? bundlePath.symlinkDestination() {
+            possibleSettingsPaths = [
+                symlink + relativePath
+                ] + possibleSettingsPaths
```

Comment:
Do you demand other mint supporting repositories to manually follow symlink?
As I suggested in https://github.com/yonaskolb/Mint/issues/53 , it should be more simple.

Reply:
No, each library shouldn't have to do this. Replying in other issue